README.md

```diff
@@ -326,7 +335,7 @@ int main(int argc, char* argv[]) {
 To compile the program using `g++` run:
 
 ```sh
-g++ -std=c++14 -DNDEBUG -O3 -I/usr/local/include -L/usr/local/lib -lkahypar -L/path/to/boost/lib -I/path/to/boost/include -lboost_program_options program.cc -o program
```

Comment:
Hm... if boost is installed system-wide that might work. But in case the MINI_BOOST option is used, one might still need so specify these paths, right?